setup.cfg

```diff
@@ -28,7 +28,7 @@ classifiers =
 include_package_data = True
 python_requires = >=3.9
 install_requires =
-  spacy>=3.1.0,<3.4.0
+  spacy>=3.0.0,<3.6.0
```

Comment:
Why are upper bounding the Spacy version? Spacy is already on 3.7 according to PyPi.

Reply:
I will remove this -- I think it was originally there when I was experimenting with 3.12 (and associated issues with spacy/textblob/etc.)